ci/test/slurm

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
removing this would help speedup the test in travis. Mocking the subprocess calls is enough to test that we're doing things right